command/helpers.go

```diff
@@ -0,0 +1,45 @@
+package command
```

Comment:
I copied this file name from nomad https://github.com/hashicorp/nomad/blob/main/command/helpers.go 

I wanted a place for common helpers that were used in various files